### PR TITLE
updating to correct for potentially offensive team pic

### PR DESCRIPTION
**Pull request by jendefig**
Updated team pic and subtitle to correct for the potentially offensive hand signals that @casperdcl pointed out this morning in July Heartbeat.  I used "obscured" and the cool emoji instead of "censored" per Slack convo, because I didn't want it to look like @mike0sv and @mvshmakov  did anything wrong because they didn't.  😊 

Also I widened the Streamlit pic while I was there.

### CODE REVIEW

File: content/blog/2021-07-16-july-21-dvc-heartbeat.md [hunk 1]
```diff
@@ -89,7 +90,7 @@ It's exciting to see that our tools are
 helpin<span style="color:purple">**g**</span> these companies in their MLOps
 workflows! 🎉
 
-![](/uploads/images/2021-07-16/job-descriptions.png)
+![](/uploads/images/2021-07-16/job-descriptions.png ')
```
Comment (shcheklein) [inline content/blog/2021-07-16-july-21-dvc-heartbeat.md:93]: a bug?

File: content/blog/2021-07-16-july-21-dvc-heartbeat.md [hunk 2]
```diff
@@ -47,8 +47,9 @@ a model, tracking changes to data and model, dvc
 gap in visualizations using <span style="color:purple">**S**</span>treamlit. You
 won't want to miss this one!
 
-![DVC and Streamlit](/uploads/images/2021-07-16/streamlit2.png) _DVC + Streamlit
-= ♥️! [Source link](https://www.sicara.ai/blog/dvc-streamlit-webui-ml)_
+![DVC and Streamlit](/uploads/images/2021-07-16/streamlit2.png '=700') _DVC +
```
Comment (shcheklein) [inline content/blog/2021-07-16-july-21-dvc-heartbeat.md:50]: where do we get this image from initially?

File: content/blog/2021-07-16-july-21-dvc-heartbeat.md (hunk 1 above)
Comment (jendefig) [reply content/blog/2021-07-16-july-21-dvc-heartbeat.md:93]: Yes.

File: content/blog/2021-07-16-july-21-dvc-heartbeat.md (hunk 2 above)
Comment (jendefig) [reply content/blog/2021-07-16-july-21-dvc-heartbeat.md:50]: From here 👇 Line 52. The blog post.

Comment (shcheklein) [reply content/blog/2021-07-16-july-21-dvc-heartbeat.md:50]: Got it, thanks. Just a quick note  - stretching images means they look a bit blurry. I hope it's fine in this case.

**Review by shcheklein — APPROVED** [review]

**Pull request merged**